x-pack/plugins/reporting/server/export_types/csv_searchsource/types.d.ts

```diff
@@ -11,7 +11,8 @@ import type { BaseParams, BasePayload } from '../../types';
 export type RawValue = string | object | null | undefined;
 
 interface BaseParamsCSV {
-  browserTimezone: string;
+  browserTimezone: string; // used for formatting timestamps
+  version: string; // version of the last supported release in case it needs migration for forward-compatibility
```

Comment:
could we rather put this on Report options and store it for every report type (inside `enqueueJob`?) and then make sure its sent over from client to server inside `reporting_api_client` `createReportJob` ? 

i think we can make this a general implementation for all report types and its going to be simpler.